src/main/java/core/basesyntax/Buldozer.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public class Buldozer extends Machines{
+    public String doWork() {
```

Comment:
Remember to use `@Override` annotation.
```suggestion
    @Override
    public void doWork() {
```